src/main/java/com/premiumminds/oidc/nimbus/NimbusOIDCTokenManagerBuilder.java

```diff
@@ -289,6 +294,21 @@ public NimbusOIDCTokenManagerBuilder expireThreshold(int expireThreshold) {
         return this;
     }
 
+    /**
+     * Set headers for the request to the token endpoint.
+     * <p>
+     * Default: empty map
+     *
+     * @param headers
+     *         map of header (name: values)
+     * @return
+     */
+    public NimbusOIDCTokenManagerBuilder headers(Map<String, List<String>> headers) {
```

Comment:
Maybe add another helper for the builder, like `header(String name, String value)`, this way the user will not need to create a Map.

Reply:
Done. I added two helpers. One with single header value and another with a list of values to be consistent with the method with a map param